mudbuild/settings.py

```diff
@@ -52,6 +53,11 @@
     'rest_auth.registration',
```

Comment:
we are now adding the installed apps as and when they are used?

Reply:
Looks good.